plutus-pab/src/Plutus/PAB/Webserver/Server.hs

```diff
@@ -50,14 +50,15 @@ import           Servant                         (Application, Handler (Handler)
                                                   hoistServer, serve, serveDirectoryFileServer, (:<|>) ((:<|>)))
 import qualified Servant
 import           Servant.Client                  (BaseUrl (baseUrlPort), ClientEnv)
+import           Wallet.Emulator.Wallet          (WalletNumber)
 
 asHandler :: forall t env a. PABRunner t env -> PABAction t env a -> Handler a
 asHandler PABRunner{runPABAction} = Servant.Handler . ExceptT . fmap (first mapError) . runPABAction where
     mapError :: PABError -> Servant.ServerError
     mapError e = Servant.err500 { Servant.errBody = LBS.pack $ show e }
 
 type CombinedAPI t =
-      API (Contract.ContractDef t) Integer
+      API (Contract.ContractDef t) WalletNumber
```

Comment:
What's the reason for using `WalletNumber` instead of `Wallet` in the API? If I understand this correctly then we can't generate a new wallet and use it for contract instances

Reply:
`WalletNumber` should only be used with plutus playground api's. I think I'm getting confused which parts are used for the plutus playground and which parts aren't.

Reply:
Ah right. The playground doesn't have a wallet API (The only thing it sends back to the server are complete simulations, it's not interactive)

Reply:
So I changed it to `WalletId`, but nothing broke! Don't we have any tests for this API?

Reply:
It _used_ to be tested by tested by a test in `plutus-pab` (`test/full/Plutus/PAB/CliSpec.hs`); but I actually ended up just hard-coding the wallet there. That particular test could be changed, though, if you wished it.